catalog/Dockerfile

```diff
@@ -16,18 +16,23 @@ RUN ln -sf /dev/stdout /var/log/nginx/access.log && \
 COPY nginx.conf /etc/nginx/nginx.conf
 COPY nginx-web.conf /etc/nginx/conf.d/default.conf
 
+ARG NGINX_STATIC_DIR=/usr/share/nginx/html
 # Copy pre-built catalog assets to nginx
-RUN rm -rf /usr/share/nginx/html
-COPY build /usr/share/nginx/html
+RUN rm -rf $NGINX_STATIC_DIR
+COPY build $NGINX_STATIC_DIR
 
 # Copy config file
 COPY config.json.tmpl config.json.tmpl
 
+# XXX: use subdir in /tmp?
```

Comment:
what for? do you foresee any problems here?

Reply:
it kinda feels safer, but I don't think it's really needed